
%NQSV(INFO): ------- Output from job:0000 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/core/fromnumeric.py:3432: RuntimeWarning:Mean of empty slice.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/core/_methods.py:190: RuntimeWarning:invalid value encountered in double_scalars
/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/data_multiarea/Model.py:878: FutureWarning:`rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.

real	0m26.296s
user	0m8.213s
sys	0m6.341s
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/dicthash/dicthash.py:47: UserWarning:Float too small for safe conversion tointeger. Rounding down to zero.
Traceback (most recent call last):
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1570, in savetxt
    fh.write(v)
OSError: [Errno 7] Argument list too long

During handling of the above exception, another exception occurred:

OSError: [Errno 7] Argument list too long

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/run_simulation.py", line 48, in <module>
    M.simulation.simulate()
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 325, in simulate
    self.write_spikes(spike_times_dict)
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 376, in write_spikes
    np.savetxt(fn, arr, fmt=fmt, header="sender time_ms",
  File "<__array_function__ internals>", line 180, in savetxt
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1577, in savetxt
    fh.close()
OSError: [Errno 7] Argument list too long
Traceback (most recent call last):
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1570, in savetxt
    fh.write(v)
OSError: [Errno 122] Disk quota exceeded

During handling of the above exception, another exception occurred:

OSError: [Errno 122] Disk quota exceeded

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/run_simulation.py", line 48, in <module>
    M.simulation.simulate()
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 325, in simulate
    self.write_spikes(spike_times_dict)
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 376, in write_spikes
    np.savetxt(fn, arr, fmt=fmt, header="sender time_ms",
  File "<__array_function__ internals>", line 180, in savetxt
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1577, in savetxt
    fh.close()
OSError: [Errno 122] Disk quota exceeded
Traceback (most recent call last):
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1570, in savetxt
    fh.write(v)
OSError: [Errno 7] Argument list too long

During handling of the above exception, another exception occurred:

OSError: [Errno 7] Argument list too long

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/run_simulation.py", line 48, in <module>
    M.simulation.simulate()
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 325, in simulate
    self.write_spikes(spike_times_dict)
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 376, in write_spikes
    np.savetxt(fn, arr, fmt=fmt, header="sender time_ms",
  File "<__array_function__ internals>", line 180, in savetxt
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1577, in savetxt
    fh.close()
OSError: [Errno 7] Argument list too long
Traceback (most recent call last):
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1570, in savetxt
    fh.write(v)
OSError: [Errno 122] Disk quota exceeded

During handling of the above exception, another exception occurred:

OSError: [Errno 122] Disk quota exceeded

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/run_simulation.py", line 48, in <module>
    M.simulation.simulate()
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 325, in simulate
    self.write_spikes(spike_times_dict)
  File "/sqfs2/cmc/1/home/u6b671/workspace/nest-gpu/ngpu_multi_area_model_simulation/multi-area-model-ngpu/multiarea_model/simulation.py", line 376, in write_spikes
    np.savetxt(fn, arr, fmt=fmt, header="sender time_ms",
  File "<__array_function__ internals>", line 180, in savetxt
  File "/opt/pyenv/versions/3.10.7/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1577, in savetxt
    fh.close()
OSError: [Errno 122] Disk quota exceeded
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[2276,1],0]
  Exit code:    1
--------------------------------------------------------------------------

real	32m5.804s
user	118m0.266s
sys	3m14.876s

%NQSV(INFO): ------- Output from job:0001 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
Currently Loaded Modulefiles:
 1) nvhpc/22.2   3) openmpi/4.0.5cu11.6    
 2) cuda/11.6    4) BaseGPU/2022(default)  

%NQSV(INFO): ------- Output from job:0002 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
Currently Loaded Modulefiles:
 1) nvhpc/22.2   3) openmpi/4.0.5cu11.6    
 2) cuda/11.6    4) BaseGPU/2022(default)  

%NQSV(INFO): ------- Output from job:0003 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
Currently Loaded Modulefiles:
 1) nvhpc/22.2   3) openmpi/4.0.5cu11.6    
 2) cuda/11.6    4) BaseGPU/2022(default)  

%NQSV(INFO): ------- Output from job:0004 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
Currently Loaded Modulefiles:
 1) nvhpc/22.2   3) openmpi/4.0.5cu11.6    
 2) cuda/11.6    4) BaseGPU/2022(default)  

%NQSV(INFO): ------- Output from job:0005 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
Currently Loaded Modulefiles:
 1) nvhpc/22.2   3) openmpi/4.0.5cu11.6    
 2) cuda/11.6    4) BaseGPU/2022(default)  

%NQSV(INFO): ------- Output from job:0006 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
Currently Loaded Modulefiles:
 1) nvhpc/22.2   3) openmpi/4.0.5cu11.6    
 2) cuda/11.6    4) BaseGPU/2022(default)  

%NQSV(INFO): ------- Output from job:0007 -------
Loading BaseGPU/2022
  Loading requirement: nvhpc/22.2 cuda/11.6 openmpi/4.0.5cu11.6
Currently Loaded Modulefiles:
 1) nvhpc/22.2   3) openmpi/4.0.5cu11.6    
 2) cuda/11.6    4) BaseGPU/2022(default)  
